server/centralized.go

```diff
@@ -226,6 +236,16 @@ func (c *Centralized) batchReads() {
 				c.log.Trace.Printf("Read: add to batch, size=%v\n", len(c.ReadBatch))
 			}
 			continue
+		case <-tick:
+			if len(c.ReadBatch) > 0 {
+				for len(c.ReadBatch) < config.ReadBatch {
```

Comment:
Shard currently expects that a batchRead will have exactly `readBatch` requests in it. - at least, that gets used to size the buffer given to pird, so the read is going to occur at that level for those non-existent reads anyway. having that happen at the leader rather than each shard maybe slightly obfuscates the individual user when the app is in a degenerate case with only one or two active clients. No strong feeling on whether this should live in the current thing called centralized vs shard, but it will need to happen in one of the two.